bolt-aws-lambda/src/main/java/com/slack/api/bolt/aws_lambda/SlackApiLambdaHandler.java

```diff
@@ -30,6 +30,10 @@ public SlackApiLambdaHandler(App app) {
         this.requestParser = new SlackRequestParser(app.config());
     }
 
+    protected App getApp() {
```

Comment:
I think a Javadoc explaining the purpose of the method would be useful. I was also thinking if it makes sense to call this method just `app` to be consistent with `App` own naming `App#config`, `App#status`, etc.

Reply:
okay, i'm fine with it.